test/test_bug3473.m

```diff
@@ -0,0 +1,95 @@
+function test_bug3453
+%filename = '/home/common/matlab/fieldtrip/data/ftp/tutorial/epilepsy/case1/ctf_data/case1.pos';
+filename = 'D:\Development\Data\case1\ctf_data/case1.pos';
+%% Individualizing a template volume conduction to on the basis of surface information
+%
+%% # Introduction
```

Comment:
please remove the lengthy comments, unless you think that it is relevant for understanding the script. Better is to point from the script to the corresponding page on the website.

Reply:
The lengthy comments have been removed.